libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/maneuver/ManeuverComponent.kt

```diff
@@ -55,3 +60,5 @@ class ManeuverComponent(
         }
     }
 }
+val Int.toDp: Int
+    get() = (this / Resources.getSystem().displayMetrics.density).toInt()
```

Comment:
Let's remove this if not intentional

Reply:
oops. That's what I get for splitting work between a Friday and a Monday.